bigquery/samples/tests/test_client_query_dry_run.py

```diff
@@ -13,10 +13,13 @@
 # limitations under the License.
 
 
-from .. import update_routine
+from .. import client_query_dry_run
 
 
-def test_update_routine(client, routine_id):
+def test_client_query_dry_run(capsys, client):
 
-    routine = update_routine.update_routine(client, routine_id)
-    assert routine.body == "x * 4"
+    query_job = client_query_dry_run.client_query_dry_run(client)
+    out, err = capsys.readouterr()
+    assert "This query will process 0 bytes." not in out
```

Comment:
I got confused by this test.

I'd very much like to see a positive assertion that a non-zero number is present. `not in` could happen for many different reasons.

```suggestion
    assert "This query will process" in out
    assert query_job.total_bytes_processed > 0
```